vyxal/main.py

```diff
@@ -26,7 +26,7 @@
     j    Print top of stack joined by newlines on end of execution
     L    Print top of stack joined by newlines (Vertically) on end of execution
     s    Sum/concatenate top of stack on end of execution
-    M    Make implicit range generation start at 0 instead of 1
+    M    Make implicit range generation start at 0 instead of 1 (and while loop counter too)
```

Comment:
Maybe change to `M Make implicit range generation and the while loop counter start at 0 instead of 1

Reply:
`Make implicit range generation and loop counter start at 0 instead of 1`